src/utils/os.h

```diff
@@ -381,7 +381,6 @@ int os_strcasecmp(const char *s1, const char *s2);
  */
```

Comment:
The commit message is confusing, the subject talks about partial revert of one commit, and the commit message body talks about another revert. Is this one now reverting a revert?
Could you reword the commit message and subject? Please put in the subject a short note what this is fixing, and in the commit message more explanation what is being changed.